Doc improvement (#52)
Unity-Technologies/com.unity.mobile.notifications: com.unity.mobile.notifications/Documentation~/index.md

# Introduction

> Mobile Notification package adds support for scheduling local one-time or repeatable notifications on Android and iOS platforms.

Reviewer: ```suggestion
The Unity Mobile Notifications package adds support for scheduling local one-time or repeatable notifications on Android and iOS.
```

### Requirements

- Compatible with Unity 2018.3 or above.
- Compatible with Android 4.4+ (API 19) and iOS 10.0+.

### Supported features

> The runtime notification APIs are split into two parts for both Android and iOS. These APIs can be used to schedule and manage notifications as listed blow:

Reviewer: ```suggestion
The runtime notification APIs are split into two parts for both Android and iOS. These APIs can be used to schedule and manage notifications as listed below:
```

*   Schedule local one-time or repeatable notifications.
*   Cancel already displayed and upcoming (scheduled) notifications.
*   Android:

>     *   Create and modify notification channels (categories) on Android Oreo and above.

Reviewer: ```suggestion
    *   Create and modify notification channels (categories) on Android 8.0 (Oreo) and above.
```

Reviewer: Just to make this easier to understand for people who might not be as familiar with Android codenames. Do we need to call out an Android API level here as well?
